### PR TITLE
Expressed neoantigens and other minor changes

### PR DESCRIPTION
**Pull request by tavinathanson**
In this PR:
- Add an `only_expressed` flag to `load_neoantigens` that uses isovar to grab protein sequences from RNA reads, and then sends those sequences through `mhctools`. CC @iskandr; am I doing anything obviously wrong?
- Add a `mannwhitneyu` test, and set two-sided to be the default.
- Return plot objects in plotting functions.

Note that I added a bunch of variables to `Cohort` that I'm not currently using anywhere, mostly for consistency. I need to find a better way to handle the `self.tumor_rna_bam_file_format_func(tumor_rna_bam_id)` situation; it's confusing that these format functions take in different sets of parameters.

### CODE REVIEW

**Comment by arahuja** [comment]
> self.tumor_rna_bam_file_format_func(tumor_rna_bam_id) situation; it's confusing that these format functions take in different sets of parameters.

Yea, perhaps we can have more consistency there. This is really just an artifact of not having the `sample_id` with the RNA files originally so they were run with RNA ID that was available at the time. We could just rename the files to something more sane.

File: cohorts/load.py [hunk 1]
```diff
@@ -263,19 +292,89 @@ def _load_single_sample_neoantigens(self, sample_idx, variant_type, merge_type,
...
+                    MAX_REFERENCE_TRANSCRIPT_MISMATCHES
+                )
+
+                # To ensure that 8-11mers overlap substitutions, we need at least this
+                # sequence length: (11 * 2) - 1
+                # Example:
+                # 123456789AB
+                #           123456789AB
+                # AAAAAAAAAAVAAAAAAAAAA
+                protein_sequence_length = (11 * 2) - 1
```
Comment (tavinathanson) [inline cohorts/load.py]: I guess this should use the max peptide length vs. assuming 8-11mers

**Comment by tavinathanson** [comment]
@arahuja I think this repo should work with RNA IDs that are != DNA IDs, though?

**Comment by tavinathanson** [comment]
Going to open a separate PR with the latest changes.

**Comment by iskandr** [comment]
My only suggestion would be to maybe increase the minimum MAPQ to 1 and if you're ultimately using the counts of neoepitopes with >= 3 spanning reads then this will be very sensitive to the degree of exonic coverage. Maybe normalize by number of reads mapping to exons?

**Pull request closed without merging**